Dashboard/Shared/DashboardPostFooter/index.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { PropTypes } from 'react';
+import {
+  Button,
+  Text,
+  WarningIcon,
+} from '@bufferapp/components';
+import {
+  borderWidth,
+} from '@bufferapp/components/style/border';
+import {
+  mystic,
+} from '@bufferapp/components/style/color';
+
+import PostFooterDelete from '../../../ColabTool/Shared/PostFooterDelete';
+import HoverableText from '../../../HoverableText';
+
```

Comment:
Looks like these styles will be mostly the same for the collabTool posts and Dashboard posts - curious how it would feel to maybe move this styles to a config file of some sort so we can just access them from both places? 😄